### PR TITLE
update input.txt

### PR DESCRIPTION
**Pull request by sevenseasofbri**

### CODE REVIEW

File: text-ui-test/input.txt [hunk 1]
```diff
@@ -1,5 +1,12 @@
...
-help
+tutorial circuit
+template lc
+set v 220
+set c 200
+set l 750
+add parallel l 200
+calc current
+exit
 template rc
```
Comment (hughjazzman) [inline text-ui-test/input.txt:3]: actually this was the old tutorial commands, can remove these then

Comment (sevenseasofbri) [reply text-ui-test/input.txt:3]: wdym?

Comment (hughjazzman) [reply text-ui-test/input.txt:3]: lines 10 to 16 were the tutorial commands previously. see the `exit` command?

Comment (sevenseasofbri) [reply text-ui-test/input.txt:3]: we need to check the circuit tutorial right

Comment (hughjazzman) [reply text-ui-test/input.txt:3]: yes I know. but we have 2 sets of tutorial commands for tutorial circuit now.

**Review by hughjazzman — APPROVED** [review]
LGTM

**Pull request merged**